crates/holochain_types/src/app/app_manifest/app_manifest_v0.rs

```diff
@@ -54,6 +54,20 @@ pub struct AppManifestV0 {
     #[serde(default)]
     #[builder(default)]
     pub allow_deferred_memproofs: bool,
+
+    /// (Optional) URL of a bootstrap server to use for all Spaces created
+    /// for Cells created for this app. If not provided here, the conductor's
+    /// default bootstrap server will be used.
+    #[serde(default)]
+    #[builder(default)]
+    pub bootstrap_url: Option<String>,
+
+    /// (Optional) URL of a signal server to use for all Spaces created
```

Comment:
```suggestion
    /// URL of a signal server to use for all Spaces created
```

Just to be consistent with other docstrings here and above.